map/framework.cpp

```diff
@@ -1073,12 +1073,9 @@ bool Framework::Search(search::SearchParams const & params)
 
 bool Framework::SearchInDownloader(DownloaderSearchParams const & params)
 {
-  // @TODO(bykoianko) It's necessary to implement searching in Storage
-  // for group and leaf mwms based on country tree.
+  static const int8_t kEngIndex = StringUtf8Multilang::GetLangIndex("en");
```

Comment:
с auto будет не так быстро

> On Mar 23, 2016, at 19:41, Vladimir Byko-Ianko <notifications@github.com> wrote:
> 
> In map/framework.cpp <https://github.com/mapsme/omim/pull/2445#discussion_r57193263>:
> 
> > @@ -1073,12 +1073,9 @@ bool Framework::Search(search::SearchParams const & params)
> >  
> >  bool Framework::SearchInDownloader(DownloaderSearchParams const & params)
> >  {
> > -  // @TODO(bykoianko) It's necessary to implement searching in Storage
> > -  // for group and leaf mwms based on country tree.
> > +  static const int8_t kEngIndex = StringUtf8Multilang::GetLangIndex("en");
> Не поменяется. Пускай будет статик, мы много где так оптимизируем.
> 
> static оставил.
> 
> А если вдруг поменяется, то пользователи со старыми mwm-ками очень сильно удивятся и останутся без английского.
> 
> В общем - по мне так оба варианта хороши :)
> Сейчас в (#2458 <https://github.com/mapsme/omim/pull/2458>) сделал auto.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2445/files/efe659814dd4aa523694b554b735bb07fc3be888#r57193263>